engine/access/rpc/backend/transactions/transactions.go

```diff
@@ -266,19 +257,77 @@ func (t *Transactions) SendRawTransaction(
 }
 
 func (t *Transactions) GetTransaction(ctx context.Context, txID flow.Identifier) (*flow.TransactionBody, error) {
-	// look up transaction from storage
 	tx, err := t.transactions.ByID(txID)
-	txErr := rpc.ConvertStorageError(err)
+	if err == nil {
+		return tx, nil
+	}
+
```

Comment:
```suggestion
	}
```
Keeping error handling block together